frontend/packages/console-app/src/components/quick-starts/utils/quick-start-utils.ts

```diff
@@ -1,21 +1,10 @@
-import { mockQuickStarts, mockStatus, mockPrerequisiteStatus } from './quick-start-mocks';
-import { QuickStartCatalogItem, QuickStartItem } from './quick-start-typings';
+import { QuickStart, QuickStartStatus } from './quick-start-types';
+import { mockQuickStarts } from './quick-start-mocks';
 
-const getQuickStarts = (): QuickStartItem[] => {
-  return mockQuickStarts;
-};
+export const getQuickStarts = (): QuickStart[] => mockQuickStarts;
 
-export const getQuickStartsWithStatus = (): QuickStartCatalogItem[] => {
-  const items = getQuickStarts();
-  return items.map((item) => {
-    return {
-      ...item,
-      ...mockStatus[item.name],
-      unmetPrerequisite: mockPrerequisiteStatus[item.name],
-    };
-  });
-};
+export const getQuickStart = (quickStartID: string): QuickStart =>
+  mockQuickStarts.find((qs) => qs.id === quickStartID);
```

Comment:
just curious , will it be mock only for 4.6?

Reply:
Yes, it's almost a mock only but with a name that does not say mock. LOL. We have a separate PR to add final quick start tour data.